server/build.gradle.kts

```diff
@@ -57,6 +58,8 @@ dependencies {
     implementation(projects.xefCore)
     implementation(projects.xefLucene)
     implementation(projects.xefPostgresql)
+    implementation("io.ktor:ktor-server-core-jvm:2.3.4")
```

Comment:
Dependencies should be added in toml file

Reply:
Idk honestly know where this time comes from. We have these dependencies already from the toml config. I'll remove it.